examples/alignment/tracking/job.json.templ

```diff
@@ -0,0 +1,19 @@
+{
+    "detector": "{{ detector }}",
+    "run_number" : {{ run_number }},
```

Comment:
This could be extracted via the filename from job.input_files['events'][0]. Would be nice if we could generalize this to run several different runs simultaneously if we want.

Reply:
by simultaneously I mean in the same jobs.json file.

Reply:
The run number is present in the event header as well but getting it requires opening the file and reading the first event. This functionality could be easily added to one of our tools if needed (like the LCIO command line tool). I don't think parsing file names is a great idea, since this is brittle and depends on whoever named the file to follow some convention, which they may not have.

Reply:
you can use `dumpevent` from lcio to get the run number used for MC files. We typically use data in alignment studies, and usually the run number will be in the file name.

Reply:
I implemented @cbravo135 's idea to use dumpevent in the jinja2 filter function. 8edc3bfae18519a672304f62da173ac127ba2ac6

I have tested this with one MC file but if it doesn't work generally, we can fall back to manually defining the run number like before.